OpenOversight/app/utils.py

```diff
@@ -29,40 +29,41 @@ def filter_by_form(form, officer_query):
     officer_query = officer_query.filter(db.or_(db.and_(Officer.birth_year <= min_birth_year,
                                                         Officer.birth_year >= max_birth_year),
                                                 Officer.birth_year == None))
+    # This handles the sorting upstream of pagination and pushes officers w/o tagged faces to the end of list
+    officer_query = officer_query.outerjoin(Face).order_by(Face.id)
```

Comment:
The only thing guaranteed to be unique is Officer.id, so whatever else we sort by, that should be the outermost sort to break ties.